jwe/aesgcm.go

```diff
@@ -0,0 +1,92 @@
+package jwe
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"errors"
+)
+
+var (
+	ErrInvalidKeySize            = errors.New("invalid key size")
+	ErrInvalidTagSize            = errors.New("invalid tag size")
+	ErrInvalidNonceSize          = errors.New("invalid nonce size")
+	ErrUnsupportedEncryptionType = errors.New("unsupported encryption type")
+)
+
+const TagSizeAESGCM = 16
+
+type EncryptionType string
+
+var A256GCM = EncryptionType("A256GCM")
```

Comment:
Should probably be EncryptionTypeA256GCM

Reply:
Renamed